src/UserProfile.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { Card, CardContent, CardHeader, Avatar, Typography, Button } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { useHistory, useParams } from 'react-router-dom';
+
+const useStyles = makeStyles((theme) => ({
+  profilePage: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100vh',
+  },
+  profileCard: {
+    maxWidth: 400,
+    width: '100%',
+  },
+  cardHeader: {
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.primary.contrastText,
+  },
+  logoutButton: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+function Profile() {
+  const classes = useStyles();
+  const history = useHistory();
+  const { userId } = useParams();
+
+  // Mock user data
+  const mockUserData = {
+    1: {
+      name: 'John Smith',
+      username: 'john123',
+      email: 'john@example.com',
+      bio: 'The song I am loveing this week is Munekita.',
+      avatar: 'https://via.placeholder.com/150',
+    },
+  
+  };
```

Comment:
This is fine for now, but we'll probably want to mock using `mirage.js` in the future so that we at least have the code set up for making the API call to the backend. Masooma wrote some docs here: https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/main/docs/DOCUMENTATION.md#mirage-library-for-mocking-backend